stacks/jhipster-online/2.33.0/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM quay.io/devfile/universal-developer-image:latest
```

Comment:
Can we pin this to a hash instead of `latest`?

Reply:
Hi! Okey, the change on this [commit](https://github.com/devfile/registry/pull/480/commits/9443741521f808f2fd3bb259c55902746be0e160).